.github/workflows/test-rc-release.yaml

```diff
@@ -0,0 +1,152 @@
+---
+name: test-rc-release
+
+on:  # yamllint disable-line rule:truthy
+  workflow_dispatch:
+    inputs:
+      rc_testing_branch:
+        # If a branch is given, the workflow will use it for deployment and testing.
+        # If no branch is provided, the workflow will create a new rc testing branch
+        # for deployment and testing.
+        description: |
+          rc_testing_branch: existing testing branch
+          (One and only one of rc_testing_branch and issue_url is required.)
```

Comment:
```suggestion
          (Either rc_testing_branch or issue_url is required. You cannot give both.)
```

Reply:
Thanks for your suggestion! This is addressed on https://github.com/astronomer/astronomer-providers/pull/1135